libraries/AP_HAL_ChibiOS/hwdef/MatekF405-Wing/hwdef.dat

```diff
@@ -199,6 +199,7 @@ define GENERATOR_ENABLED 0
 define AC_OAPATHPLANNER_ENABLED 0
 define PRECISION_LANDING 0
 define HAL_BARO_WIND_COMP_ENABLED 0
-define HAL_RUNCAM_ENABLED 0
```

Comment:
This should be a separate PR IMO.  They should come in on their merits rather than because other things have been disabled in the same PR.  I'd approve that PR, BTW.

Reply:
In my defence I am literally copying what tridge did to disable these!